File: Java/src/Remove Duplicates from Sorted Array.java
public class Solution {
    public int removeDuplicates(int[] A) {
        int length = A.length;
        if (length < 2) {
            return length;
        }

        int count = 0;
        for (int i = 1; i < length; i++ ){
            if (A[i] == A[i-1]){
                count++;
            }
            else {
                A[i-count] = A[i];
            }
        }
        return length - count;
    }
}

File: Python/Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, A):
        if not A:
            return 0
        if len(A) == 1 :
            return 1
        count = 0
        for i in range(1, len(A)):
            if A[i] == A[i-1]:
                count = count + 1
            else:
                A[i-count ] = A[i]
        length = len(A) - count     
        return length

